patches/content-browser-file_system_access-file_system_access_handle_base.cc.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/content/browser/file_system_access/file_system_access_handle_base.cc b/content/browser/file_system_access/file_system_access_handle_base.cc
+index f8712a1b0d14ed82faa3f313b0f9fdfccef23de9..fbe0f2fadbad346ddae0366228255394d51f6517 100644
+--- a/content/browser/file_system_access/file_system_access_handle_base.cc
++++ b/content/browser/file_system_access/file_system_access_handle_base.cc
+@@ -306,6 +306,7 @@ void FileSystemAccessHandleBase::DidCreateDestinationDirectoryHandle(
+     std::move(callback).Run(std::move(error));
+     return;
+   }
++  BRAVE_FILE_SYSTEM_ACCESS_HANDLE_BASE_DID_CREATE_DESTINATION_DIRECTORY_HANDLE
```

Comment:
`#define GetChildURL(NEW_ENTRY_NAME, DEST_URL) GetChildURL(NEW_ENTRY_NAME, DEST_URL); if (dest_url.type() != storage::FileSystemType::kFileSystemTypeTemporary ....` ?

Reply:
Done in https://github.com/brave/brave-core/pull/21134/commits/b29f3c046afc4b86ce8a0cd2388b4a747f172d75